test/built-ins/String/prototype/isWellFormed/length.js

```diff
@@ -0,0 +1,24 @@
+// Copyright (C) 2022 Jordan Harband. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-string.prototype.is-well-formed
+description: >
+  String.prototype.isWellFormed.length value and descriptor.
+info: |
+  String.prototype.isWellFormed( )
+
+  17 ECMAScript Standard Built-in Objects
+
+includes: [propertyHelper.js]
+features: [String.prototype.isWellFormed]
+---*/
+assert.sameValue(typeof String.prototype.isWellFormed, 'function');
+
+assert.sameValue(
+  String.prototype.isWellFormed.length, 0,
+  'The value of String.prototype.isWellFormed.length is 0'
+);
+
+verifyNotEnumerable(String.prototype.isWellFormed, 'length');
+verifyNotWritable(String.prototype.isWellFormed, 'length');
+verifyConfigurable(String.prototype.isWellFormed, 'length');
```

Comment:
Suggest using `verifyProperty` instead.

Reply:
I copied files initially from String.prototype.at; I’ll update this.